src/styles/variables.scss

```diff
@@ -0,0 +1,11 @@
+$active-color: #00acdc;
+$link-color: #060b35;
+$hover-title: #34568b;
+
+$moyo-blue: #00acdc;
```

Comment:
make up better name for this variable, it should show its purpose, not its content
P.S. Avoid using 'blue' in the name